feature/experimental/qos/ate_tests/qos_policy_config_test/qos_policy_config_test.go

```diff
@@ -332,33 +359,73 @@ func TestQoSInputIntfClassifierConfig(t *testing.T) {
 		desc                string
 		inputClassifierType oc.E_Input_Classifier_Type
 		classifier          string
+		classType           oc.E_Qos_Classifier_Type
+		termID              string
```

Comment:
You don't need to add fields when the values are the same across all the test cases. Just inline the values instead. Looks like you don't need fields for termID, dscpSet, targetGroup, or queueName. Also targetGroup is mispelled.

Reply:
All the variables (termID, dscpSet, targetGroup, or queueName.) are needed in the function.
Please provide the example about inlining the values

Reply:
Add the top of the test function you can add:

const (
  termID = "0",
  targetGroup = "target-group=BE1",
  queueName = "0"
)
dscpSet := []uint8{0, 1, 2, 3}

and just refer to these, which will make it clear they are _not_ parameterized per subtest -- they are always the same.